<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordEditWidget</class>
 <widget class="QWidget" name="PasswordEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>78</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="enterPasswordLabel">
     <property name="text">
      <string>Enter password:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PasswordEdit" name="enterPasswordEdit">
       <property name="accessibleName">
        <string>Password field</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="togglePasswordButton">
       <property name="accessibleName">
        <string>Toggle password visibility</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="PasswordEdit" name="repeatPasswordEdit">
       <property name="accessibleName">
        <string>Repeat password field</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="passwordGeneratorButton">
       <property name="accessibleName">
        <string>Toggle password generator</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="repeatPasswordLabel">
     <property name="text">
      <string>Confirm password:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordEdit</class>
   <extends>QLineEdit</extends>
   <header>gui/PasswordEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>enterPasswordEdit</tabstop>
  <tabstop>repeatPasswordEdit</tabstop>
  <tabstop>togglePasswordButton</tabstop>
  <tabstop>passwordGeneratorButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
